conf/ethz_euler.config

```diff
@@ -0,0 +1,51 @@
+/*
+* ------------------------------------------------------------------
+*   nf-core pipelines config file for the ETH Zurich Euler cluster
+* ------------------------------------------------------------------
+*/
+
+params {
+        config_profile_name        = 'ethz_euler'
+        config_profile_description = 'ETH Zurich Euler cluster profile -- provided by nf-core/configs'
+        config_profile_contact     = 'Joao Agostinho de Sousa (github: @jpadesousa; email: joao.agostinhodesousa@hest.ethz.ch)'
+        config_profile_url         = 'https://scicomp.ethz.ch/wiki/Euler'
+
+        max_memory = 4.TB
+        max_cpus   = 128
+        max_time   = 120.h
+        
+	igenomes_base   = '/cluster/project/igenomes'
+        igenomes_ignore = false
+}
+
+executor {
+        name                = 'slurm'
+        perCpuMemAllocation = true
+}
+
+process {
+        scratch             = 'true'
+        errorStrategy       = 'retry'
+        maxRetries          = 3
+        maxErrors           = '-1'
+        beforeScript        = 'module load eth_proxy'
+}
+
+singularity {
+        enabled    = true
+        autoMounts = true
+        cacheDir   = "${APPTAINER_CACHEDIR}"
```

Comment:
```suggestion
```

This doesn't really make sense to me - what apptainer uses `APPTAINER_CACHEDIR` for is quite different from what nextflow uses it for (e.g. for apptainer you really don't want it shared whereas for the nextflow case it makes sense to keep them shared within the process directory).

I'm also not certain it's easier to set `APPTAINER_CACHEDIR` than `NXF_SINGULARITY_CACHEDIR` and the default behaviour of keeping them in work if not set seems more sensible than what I guess will happen here if unset.

Reply:
Thank you for the input. You are right. We have already NXF_SINGULARITY_CACHEDIR  defined once the user loads the nextflow module. So this would just require and extra unnecessary step. I will remove it.